edge-util/src/Microsoft.Azure.Devices.Edge.Storage/InMemoryDbStoreProvider.cs

```diff
@@ -24,9 +106,225 @@ public void RemoveDbStore(string partitionName)
             this.partitionDbStoreDictionary.TryRemove(partitionName, out IDbStore _);
         }
 
+        public void Close()
+        {
+            this.CloseAsync().Wait();
```

Comment:
Don't do Wait().. 
Can you simply not have a sync Close() and only have CloseAsync()

Reply:
Done, made all of it async